src/main/java/com/linecorp/armeria/server/docs/EnumInfo.java

```diff
@@ -22,14 +22,15 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 
 import org.apache.thrift.meta_data.EnumMetaData;
 import org.apache.thrift.protocol.TType;
 
 class EnumInfo extends TypeInfo implements ClassInfo {
 
-    static EnumInfo of(EnumMetaData enumMetaData) {
+    static EnumInfo of(EnumMetaData enumMetaData, String packageName, Map<String, String> docStrings) {
```

Comment:
Yeah, I noticed that after leaving this comment. I still think the factory methods should require only as much as it needs.